ui/packages/app/web/src/libs/utils.ts

```diff
@@ -7,3 +7,7 @@ export const cutToMaxStringLength = (input: string, len: number): string => {
   }
   return `${input.substring(0, len)}...`;
 };
+
+export const capitalizeFirstLetter = (str: string): string => {
```

Comment:
Looks like we already have a capitalize function [here](https://github.com/parca-dev/parca/blob/main/ui/packages/shared/functions/src/index.ts#L1). will be nice to reuse that fn here

Reply:
Ah, good catch, will change it.